### DISCUSSION TITLE
Demo

### DISCUSSION THREAD

**Discussion by spalheapp** [Antoinegtir/instagram-threads#20] [Q&A] [answered]
APK demo app Threads clone?

**Comment by Antoinegtir** [comment]
> APK demo app Threads clone?

Yes for sure i will send you the apk in a few hour !

**Comment by Antoinegtir** [comment, marked as answer]
> APK demo app Threads clone?

APK is now realised you <a href="https://github.com/Antoinegtir/instagram-threads/releases/tag/0.0.2">Click here to download</a> 🎉
